### ISSUE TITLE
SPiked Squig:  Please fix the missing 5% crit

### ISSUE THREAD

**Issue by mursie** [WarEmu/WarBugs#2373]
Spiked squig should give the squig herder 5% ranged crit.  It does correctly show in the character sheet but it is not showing up in .getstats

There has been a fix pending on this for quite some time.  After many many server restarts and patches, I'm afraid this has fallen through the cracks.

Please

**Comment by Evildoor** [comment]
#2064
Do not create a new ticket in such case. If devs are not fixing something then there is a reason for that.
